src/linear_algebra/multilinear/basis.lean

```diff
@@ -0,0 +1,98 @@
+/-
+Copyright (c) 2021 Joseph Myers. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Joseph Myers
+-/
+import linear_algebra.basis
+import linear_algebra.multilinear.basic
+
+/-!
+# Multilinear maps in relation to bases.
+
+This file proves lemmas about the action of multilinear maps on basis vectors.
+
+-/
+
+open multilinear_map
+
+variables {R : Type*} {ι : Type*} {n : ℕ} {M : fin n.succ → Type*} {M₂ : Type*} {M₃ : Type*}
+variables [comm_semiring R] [add_comm_monoid M₂] [add_comm_monoid M₃] [∀i, add_comm_monoid (M i)]
+variables [∀i, module R (M i)] [module R M₂] [module R M₃]
+
+/-- Two multilinear maps indexed by `fin n.succ` are equal if they are equal when all arguments
+are basis vectors. -/
+lemma basis.ext_multilinear_fin {f g : multilinear_map R M M₂} {ι₁ : fin n.succ → Type*}
```

Comment:
This proof is much shorter if you replace `fin n.succ` with `fin n`, and then the base case of the induction is just
```lean
  { ext x,
    convert h fin_zero_elim },
```

Reply:
Done.